src/transport/SecureSession.h

```diff
@@ -98,15 +99,26 @@ class SecureSession : public Session
      *   PASE, setting internal state according to the parameters used and
      *   discovered during session establishment.
      */
-    void Activate(Type secureSessionType, const ScopedNodeId & peer, CATValues peerCATs, uint16_t peerSessionId,
-                  const ReliableMessageProtocolConfig & config)
+    void Activate(Type secureSessionType, const ScopedNodeId & peerNode, const ScopedNodeId & localNode, CATValues peerCATs,
+                  uint16_t peerSessionId, const ReliableMessageProtocolConfig & config)
```

Comment:
Again, should localNode come first, before all the peer info?

Reply:
I will update